scripts/normalize_merchant_names.py

```
@@ -0,0 +1,259 @@
+        )
+        for name, count in orphan_counts.most_common(20):
+            print(f"  {count:>4}  {name}")
+        print()
+
+    if rejected_diff_text:
+        print(
+            f"REJECTED: {len(rejected_diff_text)} candidates where sibling "
+            "name differs by more than casing (likely data-quality issue, "
+            "needs manual review):"
+        )
+        for old, new in Counter(rejected_diff_text).most_common(20):
```

Ruff (0.15.14)

[warning] Loop control variable `new` not used within loop body

(B007)